Added UserRepository and IUserRepository

poprostupiotrus opened the pull request:
Added UserRepository and IUserRepository

ajklis reviewed (changes requested):
IUserRepo and UserRepo were already created, delete them

poprostupiotrus commented:
I deleted IUserRepo and UserRepo from project and also deleted not used namespaces in Program.cs. Please, accept this pull request or you will have big problems habibi.
